### DISCUSSION TITLE
Embed PDF?

### DISCUSSION THREAD

**Discussion by root9191** [requarks/wiki#4928] [Help / Questions]
Hello,
is it possible to embed a PDF on my Wiki Page?
I tried this via iFrame but it don't work, i still get asked for Downloading, instead of embedding:

```
<p>This is a to use an iFrame to display an WikiJS internal (asset) PDF</p>
<div>
<iframe src="/misc/kxk.pdf#zoom=150&page=70" width=75% height="600"></iframe>
</div>
```

**Comment by root9191** [comment]
No one?

**Comment by CodeByChaim** [comment]
Hi, I am running into the issue, did you find a solution to embedding PDF? I also looking for embedding PowerPoint presentation
Thnx

**Comment by NGPixel** [comment]
Trying disabling "Force Download of Unsafe Extensions" under Admin > Security.

**Reply by DarickBrokaw** [reply]
Nope, this did not work for me. Thank for the idea anyways.

**Reply by root9191** [reply]
How do you embed the PDF File?

**Comment by CodeByChaim** [comment]
thanks 🙏
________________________________
